### PR TITLE
 On branch feature/carol-assignment-branch

### PR DESCRIPTION
**Pull request by uxbycarol**
push test

### CODE REVIEW

**Comment by uxbycarol** [comment]
Hi buhodev,  I would like to include the check icon in mi design as in the image. While I could add the SVG manually, I prefer to integrate it as a icon component.

Could you please provide me with the code for the icon component?.

Thank you for your assistance.

image

**Comment by buhodev** [comment]
We're not using a component for the icons, we're using a preprocessor that creates the html at build time (https://github.com/unplugin/unplugin-icons). But a simple component for the Check icon will be this:

```svelte
<script lang="ts">
	let className: string | undefined = 'size-5'
	export { className as class };
</script>

<!-- SVG of the "Check" Mini icon copied from https://heroicons.com -->
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" class={className}>
	<path
		fill-rule="evenodd"
		d="M16.704 4.153a.75.75 0 0 1 .143 1.052l-8 10.5a.75.75 0 0 1-1.127.075l-4.5-4.5a.75.75 0 0 1 1.06-1.06l3.894 3.893 7.48-9.817a.75.75 0 0 1 1.05-.143Z"
		clip-rule="evenodd"
	/>
</svg>
```

I'd recommend to just use the svg icon and modify the properties in the html rather than use a component

**Comment by uxbycarol** [comment]
Thanks, I followed your recommendation. The assignment is ready for review

File: src/app.css [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+    --theme-rounded-container: 16px;
+    --theme-border-base: 1px;
+    --on-primary: #ffffff;
+    --on-secondary: #ffffff;
+    --on-tertiary: #ffffff;
+    --on-success: #000000;
+    --on-warning: #000000;
+    --on-error: #000000;
+    --on-surface: #ffffff;
+    --color-primary-50: #fdf1e9;
```
Comment (buhodev) [inline src/app.css]: Tailwind already generates all the utilities based on the colors that you added from line 18 until line 47. They're defined in the `tailwind.config.js` file. If you're not using the css variables somewhere else, I'd remove them from here

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+
+<main class="flex h-full items-center justify-center bg-surface-900">
+	<Feature
+		title="Optimize Your Cloud Spend with Granular Cost Management"
+		description="Gain complete visibility and control over your Kubernetes costs with our comprehensive cost management features. Our platform provides you with the tools you need to:"
+		ctaText="Book a free demo"
+		ctaLink="#"
+		imageSrc="/cost-management-image.png"
+		imageAlt="Cost Management Feature"
+		><slot>
```
Comment (buhodev) [inline src/routes/+page.svelte]: It's not necessary to wrap the markup inside of a `<slot>` tag here. You can safely remove the `slot`

File: src/routes/feature.svelte [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    export let imageSrc;
+    export let imageAlt;
+  </script>
+<section >
+    <div class="w-full h-full inset-0 absolute blur-3xl pointer-events-none" style="background:linear-gradient(202.72deg, rgba(237, 78, 139, 0.1) 14.76%, rgba(152, 103, 240, 0.09) 34.37%, rgba(152, 103, 240, 0) 86.62%, rgba(0, 0, 255, 0.1) 100%)"></div>
+    <div class="flex max-md:flex-col items-center px-6 lg:px-12 py-20 gap-8 max-w-7xl m-auto z-10">
+        <div class="flex flex-col basis-3/5 gap-3 md:gap-6">
+            <h2 class="text-2xl xl:text-4xl font-bold text-white">{title}</h2>
+            <p class="text-pretty text-base xl:text-xl text-gray-300">{description}</p>
+          <slot></slot>
```
Comment (buhodev) [inline src/routes/feature.svelte]: Here it's ok to keep the `<slot>` tag

**Review by buhodev — CHANGES_REQUESTED** [review]
Good job, Carol. Please run `bun run format` inside of your terminal to format the files after addressing the comments I left bellow, and commit the changes

File: src/routes/+page.svelte (hunk 2 above)
Comment (uxbycarol) [reply src/routes/+page.svelte]: Oh, thanks

**Pull request merged**